langchain/src/vectorstores/milvus.ts

```diff
@@ -223,6 +232,7 @@ export class Milvus extends VectorStore {
       output_fields: outputFields,
       vector_type: DataType.FloatVector,
       vectors: [query],
+      filter: filterStr,
```

Comment:
Just checking - passing an empty string has the same behavior as undefined right?

Reply:
Yeah, sure.

Reply:
Hey actually it looks like Milvus just accepts a string: 

https://milvus.io/docs/search.md#Conduct-a-vector-search

I think we can just have `FilterType: string` here - if they add more complex objects in the future we can shim it.

Reply:
No problem, but I think that `FilterType` is defined as object type (and thus I created interface), I can add to base class `object | string` what do you think about that?

Reply:
@jacoblee93 I changed it.